simulator.py

```diff
@@ -12,6 +12,118 @@
 from cogent.evolve.models import JC69
 
 
+def different_tree_simulate_alignment(tree_information_list,all_trees):
```

Comment:
I see what you're doing here, but I have a couple of questions/observations.

1. Why do you want to simulate alignments with different underlying topolgies? That's a (very) different inference problem to the one we discussed previously. It's still a very interesting problem (and an important one) but I'm interested to know your aims here. I.e. what are you hoping to do with these simulated alignments. Combining different trees with heterotachy seems like an exceptionally complex place to start any analysis!

2. It's misleading to return a 'true' tree here. There is no one true tree for this alignment. For example, consider the case in which tree1 is 90% of the alignment and tree2 is 10%. If tree1 has very short branches, then there will be very few informative sites. If tree2 has useful branch lengths (e.g. ~0.2), then there will be lots of informative sites. In this case, the 'best' tree (though it's still incorrect) might be tree2, not tree1. Either way though, there is no single true tree. There is only a true pair of trees.